.clang-tidy

```diff
@@ -10,6 +10,14 @@
 #      `auto Foo() -> std::string { return ...; }`, we think the code is less
 #      readable in this form.
 #
+#  --modernize-concat-nested-namespaces: clang-tidy recommends
+#      `namespace google::cloud {}` over `namespace google { namespace cloud { } }`
+#      We think the code is more readable without nested namespaces.
```

Comment:
"We need to support C++14, which does not supported nested namespaces."  I for one think nested namespaces are more readable.

Reply:
Done